Formula/libzt.rb

```diff
@@ -0,0 +1,36 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+    :branch   => "master",
+    :revision => "9dbcc8eb45bf8863261599962c799b1de3557d14"
+
+  version "1.3.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "git", "submodule", "update", "--init"
+    system "git", "-C", "ext/lwip", "apply", "../lwip.patch"
+    system "git", "-C", "ext/lwip-contrib", "apply", "../lwip-contrib.patch"
+    system "git", "-C", "ext/ZeroTierOne", "apply", "../ZeroTierOne.patch"
```

Comment:
Why apply these patches? Why are they not applied by default?

Reply:
Good question, this is because:

(1). lwIP/lwIP-contrib: The philosophy of these projects is minimalism and our patch introduces features that are not useful to most people and would thus not be accepted into their mainline.

(2). ZeroTierOne: patches exist because they are changes that are acceptable in our SDK (libzt) but not necessarily ready for our mainline yet.

Unfortunately, this is the one change that I don't think I will be able to accommodate.

Reply:
If the features are not ready for mainline I'd rather they aren't applied in the formula.

Reply:
Sure thing. I'll work on getting the changes into the ZeroTierOne mainline.

Reply:
Incorporated changes into ZeroTierOne mainline and removed patch.